packages/blast-utilities/contract-utils.js

```diff
@@ -27,9 +27,9 @@ module.exports.CudosContract = class CudosContract {
     }
   }
 
-  async deploy(initMsg, label = this.#contractLabel) {
+  async deploy(initMsg, funds, label = this.#contractLabel) {
```

Comment:
We shouldn't force the user to pass funds when deploying a contract

Reply:
is it just me, or in this case we are still forcing the users to fund the contract every time?

Reply:
if the funds value is not set we are not funding the contract